packages/smithy-http/src/smithy_http/aio/crt.py

```diff
@@ -349,6 +349,7 @@ async def _marshal_request(
                 headers_list.append((fld.name, val))
 
         path = self._render_path(request.destination)
+        print("CRT path: ", path)
```

Comment:
```suggestion

```

Reply:
Doh - I thought I removed all of these debugs....

Reply:
There is a lint rule that we can add to catch print statements: https://docs.astral.sh/ruff/rules/#flake8-print-t20

Reply:
I did try to add `T20` to the list configured in our top level [pyproject.toml](https://github.com/smithy-lang/smithy-python/blob/develop/pyproject.toml#L54) but it didn't seem to catch usage of print, so I'm not sure what I did wrong.